frontend/src/features/archived-items/crawl-log-table.ts

```diff
@@ -0,0 +1,309 @@
+                      hour="2-digit"
+                      minute="2-digit"
+                      second="2-digit"
+                      hour-format="24"
+                    >
+                    </btrix-format-date>
+                    <btrix-format-date
+                      slot="content"
+                      date=${log.timestamp}
+                      month="long"
+                      day="numeric"
+                      year="numeric"
+                      hour="numeric"
+                      minute="numeric"
+                      second="numeric"
+                      time-zone-name="short"
+                    >
+                    </btrix-format-date>
+                  </sl-tooltip>
+                </div>
+                <div class="pr-4 text-center">
+                  <sl-tooltip
+                    class="capitalize"
+                    content=${log.logLevel}
+                    placement="bottom"
+                    @sl-hide=${stopProp}
+                    @sl-after-hide=${stopProp}
+                  >
+                    ${this.renderLevel(log)}
+                  </sl-tooltip>
+                </div>
+                <div class="whitespace-pre-wrap">${log.message}</div>
+                ${log.details.page
+                  ? html`
+                      <div class="truncate" title="${log.details.page}">
+                        ${log.details.page}
+                      </div>
+                    `
+                  : html`<div class="text-neutral-400 group-hover:text-inherit">
+                      ${noData}
+                    </div>`}
+              </div>
+            </btrix-numbered-list-item>
+          `;
+        })}
+      </btrix-numbered-list>
+
+      <btrix-dialog
+        .label=${msg("Log Details")}
+        .open=${!!this.selectedLog}
+        class="[--width:40rem]"
+        @sl-show=${stopProp}
+        @sl-after-show=${stopProp}
+        @sl-hide=${stopProp}
+        @sl-after-hide=${(e: CustomEvent) => {
+          stopProp(e);
+          this.selectedLog = null;
+        }}
+        >${this.renderLogDetails()}</btrix-dialog
+      > `;
+  }
+
+  private renderLevel(log: CrawlLog) {
+    const logLevel = log.logLevel;
+    const contextLevel =
+      contextLevelFor[log.context as unknown as CrawlLogContext] || 0;
+    const baseClasses = tw`size-4 group-hover:text-inherit`;
+
+    switch (logLevel) {
+      case CrawlLogLevel.Fatal:
+        return html`
+          <sl-icon
+            name="exclamation-octagon-fill"
+            class=${clsx(tw`text-danger-500`, baseClasses)}
+          ></sl-icon>
+        `;
+      case CrawlLogLevel.Error:
+        return html`
+          <sl-icon
+            name="exclamation-triangle-fill"
+            class=${clsx(tw`text-danger-500`, baseClasses)}
+          ></sl-icon>
+        `;
+      case CrawlLogLevel.Warning:
+        return html`
+          <sl-icon
+            name="exclamation-circle-fill"
```

Comment:
We haven't used it yet I believe, but with this many states I think the shape distinction from the info level is useful
```suggestion
            name="exclamation-diamond-fill"
```